src/redux/sagas/motions/rootMotion.ts

```diff
@@ -139,11 +137,11 @@ function* createRootMotionSaga({
     if (annotationMessage) {
       yield put(transactionPending(annotateRootMotion.id));
 
-      const ipfsHash = yield call(uploadIfpsAnnotation, annotationMessage);
+      //  const ipfsHash = yield call(uploadIfpsAnnotation, annotationMessage);
 
-      yield put(
-        transactionAddParams(annotateRootMotion.id, [txHash, ipfsHash]),
-      );
+      // yield put(
+      //   transactionAddParams(annotateRootMotion.id, [txHash, ipfsHash]),
+      // );
```

Comment:
No more IPFS for now, so this can get removed entirely

Reply:
@rdig Assuming we'll be uploading annotations to the db via block-ingestor, is it OK to use the `annotationMessage` itself as the second param to the `annotateTransaction` contract method (i.e. the metadata param)? My understanding is that generally storing strings on chain is expensive gas-wise, but since `annotateTransaction` just emits an event, is it OK to call `annotateTransaction` with the actual annotation message so it can be picked up by the block ingestor when it detects the event?

If not, we would presumably need to mutate the db from the saga, since I can't think of how else we'd get the actual annotation msg to the block ingestor.